addons/web/static/tests/views/list_view_tests.js

```diff
@@ -11943,6 +11943,8 @@ QUnit.module("Views", (hooks) => {
 
         assert.verifySteps(["write", "web_read"]);
         // select the second record (the first one is still selected)
+        // discard selection
+        await click(target, ".o_list_unselect_all");
         assert.containsNone(target, ".o_list_record_selector input:checked");
         await click(rows[0], ".o_list_record_selector input");
```

Comment:
suggestion:
```js
assert.containsOnce(target, ".o_list_record_selector input:checked");
```
and remove the click on the first row and the click you added to unselect all

This would be more representative of a real flow